src/Payments.Mvc/Controllers/InvoicesApiController.cs

```diff
@@ -51,7 +51,70 @@ public async Task<ActionResult<Invoice>> Get(int id)
 
             return invoice;
         }
-        
+
+
+        /// <summary>
+        /// Mark Invoice as Deleted.
+        /// Invoice must be in the drafted state or not paid or refunded
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns>Invoice</returns>
+        [HttpDelete("{id}")]
+        [ProducesResponseType(typeof(Invoice), 200)]
+        [ProducesResponseType(400)]
+        [ProducesResponseType(404)]
+        public async Task<ActionResult<Invoice>> Delete(int id)
+        {
+            var team = await GetAuthorizedTeam();
+            var invoice = await _dbContext.Invoices
+                .Include(i => i.Items)
+                .Include(i => i.Attachments)
+                .Include(i => i.Team)
```

Comment:
why include these, i don't see you using any of them

Reply:
Attachments doesn't do anything though.

Reply:
i wouldn't return anything with a delete